### PR TITLE
Next time husky commit style =)

### PR DESCRIPTION
**Pull request by RujaMarcello**

### CODE REVIEW

File: frontend/src/components/users-table/index.tsx [hunk 1]
```diff
@@ -67,11 +68,7 @@ const UserTable: React.FC<TableProps> = (data) => {
           };
         }}
       ></Table>
-      <Modal title="Name+Profile Picture" open={isModalOpen} onOk={handleOk} onCancel={handleCancel}>
-        <p>Some contents...</p>
-        <p>Some contents...</p>
-        <p>Some contents...</p>
-      </Modal>
+      <UserDataPopup open={isModalOpen} onOk={() => handleOk()} onCancel={() => handleCancel()} />
```
Comment (ducaOvidiu) [inline frontend/src/components/users-table/index.tsx:71]: `<UserDataPopup open={isModalOpen} onOk={handleOk} onCancel={handleCancel} />`

**Comment by ducaOvidiu** [comment]
Please change commit style is not something funny

File: frontend/src/components/bottom-menu/index.module.scss [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+          justify-content: center;
+      }
+    }
+  
+  .Spacer {
+    flex: 1;
+  }
+  
+  
+
```
Comment (ducaOvidiu) [inline frontend/src/components/bottom-menu/index.module.scss]: You don t need space here

File: frontend/src/components/superadmin-menu/index.module.scss [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+          display: flex;
+          align-items: center;
+          justify-content: center;
+      }
+    }
+  
+  .Spacer {
+    flex: 1;
+  }
+
```
Comment (ducaOvidiu) [inline frontend/src/components/superadmin-menu/index.module.scss]: Also here

File: frontend/src/components/user-menu/index.module.scss [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+          justify-content: center;
+      }
+    }
+  
+  .Spacer {
+    flex: 1;
+  }
+  
+  
+
```
Comment (ducaOvidiu) [inline frontend/src/components/user-menu/index.module.scss]: Space here is not requierd

File: frontend/src/components/users-table/index.tsx [hunk 5]
```diff
@@ -1,28 +1,19 @@
...
-  };
-
-  const handleCancel = () => {
-    setIsModalOpen(false);
-  };
-
-  const showModal = () => {
-    setIsModalOpen(true);
-  };
+  console.log(data);
```
Comment (ducaOvidiu) [inline frontend/src/components/users-table/index.tsx]: You don t need to push console.log into dev it is just for testing

File: frontend/src/pages/dashboard/all-users/all-users.tsx [hunk 6]
```diff
@@ -1,20 +1,29 @@
...
+  const getCurrentPage = (currentPage: number) => {
+    setPage(currentPage);
+  };
+
   useEffect(() => {
     const fetcher = async () => {
-      const response = await api.user.usersGet();
+      setLoading(true);
+      const response = await api.user.usersGet({ page: page }).finally(() => setLoading(false));
+      console.log(response.data);
```
Comment (ducaOvidiu) [inline frontend/src/pages/dashboard/all-users/all-users.tsx]: You have console.log(here)

**Review by ducaOvidiu — CHANGES_REQUESTED** [review]

**Review by Ovyzz — APPROVED** [review]

**Pull request merged**